### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### LINKED ISSUE #65: CobaltStrike 4.6 must be installed manually
# 💡 Summary #

CobaltStrike 4.6 has recently come out and will require the new package be downloaded from their website directly. Every subsequent update can follow the same process that is currently used (utilizing the update binary located in the /opt/cobaltstrike folder on the Kali and TeamServer images). 

## Motivation and context ##

If we don't pull the latest version of CobaltStrike directly from their website for 4.6 we will be unable to continue to use the latest features of CobaltStrike moving forward. 

## Implementation notes ##

Please see the following blog which further outlines why this is necessary (https://www.cobaltstrike.com/blog/cobalt-strike-4-6-the-line-in-the-sand/)

## Acceptance criteria ##

How do we know when this work is done?

- [ ] CobaltStrike 4.6 has been downloaded and installed on the Kali AMI
- [ ] CobaltStrike 4.6 has been downloaded and installed on the TeamServer AMI
- [ ] We can once again use the update binary located in the /opt/cobaltstrike directory to update future instances (This will be possible only after downloaded and installing CobaltStrike 4.6)

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role-with-test-user.git`](https://github.com/cisagov/skeleton-ansible-role-with-test-user.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-cobalt-strike.git ansible-role-cobalt-strike
    cd ansible-role-cobalt-strike
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role-with-test-user.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add meta/main.yml molecule/default/molecule-no-systemd.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

---

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by jsf9k** [comment]
Resolves #65.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Excellent, thanks for fixing this! 🍓

**Comment by jsf9k** [comment]
@mcdonnnj, this one is finally ready for review when you have time.

File: molecule/default/tests/test_default.py [hunk 1]
```diff
@@ -14,7 +14,11 @@
 
 @pytest.mark.parametrize(
     "f",
-    ["/root/.cobaltstrike.license", "/opt/cobaltstrike", "/opt/cobaltstrike/update"],
+    [
+        "/opt/cobaltstrike/cobaltstrike.auth",
+        "/opt/cobaltstrike",
+        "/opt/cobaltstrike/update",
```
Comment (mcdonnnj) [inline molecule/default/tests/test_default.py]: Just updating the sorting for these with the value change.

```suggestion
        "/opt/cobaltstrike",
        "/opt/cobaltstrike/cobaltstrike.auth",
        "/opt/cobaltstrike/update",
```

**Review by mcdonnnj — APPROVED** [review]
Thanks for getting this working again 💪💪💪 I had one minor suggestion but otherwise LGTM ✔

**Pull request merged**